runelite-client/src/main/java/net/runelite/client/plugins/kittentracker/KittenPlugin.java

```diff
@@ -154,14 +159,17 @@ private void getFollowerID()
 		{
 			s = "0" + s;
 		}
-
+		int previousFollowerID = followerID;
 		// followerID is the first 2 octets converted into decimals
 		followerID = Integer.parseInt(s.substring(0, 16), 2);
+		if (followerID == previousFollowerID)
```

Comment:
My commit --amend went a bit wrong there. I forgot to add these 5 lines to the commit